.ci/pytorch/multigpu-test.sh

```diff
@@ -54,6 +54,9 @@ time python test/run_test.py --verbose -i distributed/_composable/fsdp/test_full
 # Pipelining composability tests
 time python test/run_test.py --verbose -i distributed/pipelining/test_composability.py
 
+# 3D composability tests
+time python test/run_test.py --verbose -i distributed/_composable/test_composability/test_fsdp_tp_dp_composable
```

Comment:
Would this file include FSDP + TP + PP? Otherwise, would this just be 2D?

Reply:
I would include the [TP](https://github.com/pytorch/pytorch/pull/129726), [DP](https://github.com/pytorch/pytorch/pull/129727)

Reply:
So I guess this is just 2D composability for now?